src/EA.Iws.Domain.Tests.Unit/Movement/ReceiptDateTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿namespace EA.Iws.Domain.Tests.Unit.Movement
+{
+    using Core.Movement;
+    using Core.Shared;
+    using Domain.Movement;
+    using Prsd.Core;
+    using System;
+    using System.Threading.Tasks;
+    using TestHelpers.DomainFakes;
+    using TestHelpers.Helpers;
+    using Xunit;
+    public class ReceiptDateTests
+    {
+        private Movement movement;
+        private readonly Guid userId = new Guid("E45663E5-1BD0-4AC3-999B-0E9975BE86FC");
+        private static readonly Guid NotificationId = new Guid("675287E3-42D3-4A58-86D4-691ECF620671");
+        private static readonly Guid MovementId = new Guid("DD7A435F-BB74-4EA6-8680-C1811C46500A");
+        private static readonly DateTime Today = new DateTime(2017, 07, 19);
+        private static readonly DateTime PastDate = Today.AddDays(-2);
+        private static readonly DateTime FutureDate = Today.AddDays(2);
+
+        public ReceiptDateTests()
```

Comment:
You need to implement IDisposable and call SystemTime.Unfreeze() in the Dispose() method